src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,190 @@
+package duke;
+
+import duke.tasktypes.Task;
+import duke.tasktypes.TaskList;
+
+/**
+ * Deals with the interactions with the user.
+ */
+public class Ui {
+
+    public Ui() {}
+
+    /**
+     * Shows introduction paragraph and instructions to user.
+     */
+    public void showIntro() {
+        String dory = "      _\n"
+                + "     | |\n"
+                + "   __| |   ___    _ __   _   _\n"
+                + "  /    |  /   \\  | /__| | | | |\n"
+                + " |   O | |  O  | | |    | |_| |\n"
+                + "  \\__,_|  \\___/  |_|     \\__, |\n"
+                + "   ________________________/  |\n"
+                + "  (__________________________/\n"
+                + "\n"
+                + " how to use:\n"
+                + "     * type down something and i'll remember\n"
+                + "         'todo' followed by your 'task'\n"
+                + "             eg. todo go to sleep\n"
+                + "         'deadline' followed by the '/by yyyy-mm-dd'\n"
+                + "             eg. deadline finish test /by 2020-12-31\n"
+                + "         'event' followed by the '/at yyyy-mm-dd'\n"
+                + "             eg. event christmas /at 2020-10-10\n"
+                + "     * type 'list' to show everything\n"
+                + "     * type 'bye' to leave\n"
+                + "     * type 'done' followed by the task number\n"
+                + "       to mark it as done\n"
+                + "     * type 'delete' followed by the task number\n"
+                + "       to delete it from your list\n"
+                + " >";
+
+        String fish = "                              ....\n"
+                + "                             /.... \\\n"
+                + "   hi my name is dory    .-`\\ \\   `...')\n"
+                + "   and i can help you   ( o   | |      (\n"
+                + "    remember things      `-_ / /_./``-._)\n"
+                + "                             `\\___\\";
+
+        // introduction to chat bot
+        System.out.println(fish);
+        System.out.println(dory);
+    }
+
+    /**
+     * Displays deleted task.
+     * @param removed Deleted Task.
+     * @param taskList TaskList containing remaining tasks.
```

Comment:
Really nitpicking here (code too clean haha), but i think these descriptions don't need capitalisations if its not a name, its probably just deleted task right?